llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -3221,7 +3221,7 @@ void LoopVectorizationCostModel::collectLoopScalars(ElementCount VF) {
 
     // Determine if all users of the induction variable update instruction are
     // scalar after vectorization.
-    auto ScalarIndUpdate =
+    bool ScalarIndUpdate =
         llvm::all_of(IndUpdate->users(), [&](User *U) -> bool {
```

Comment:
while you are editing around here, might as well drop redundant llvm::


```suggestion
        all_of(IndUpdate->users(), [&](User *U) -> bool {
```